src/Component/Symbolizer/PropTextEditor/PropTextEditor.tsx

```diff
@@ -67,8 +95,8 @@ export class PropTextEditor extends React.Component<PropTextEditorProps> {
   }
 
   formatLabel = (label: string): string => {
-    const prefix = '\\{\\{';
-    const suffix = '\\}\\}';
+    const prefix = '\{\{';
+    const suffix = '\}\}';
```

Comment:
Please restore the double `\` here, even though they probably can be removed entirely in a separate PR.
```suggestion
    const suffix = '\\}\\}';
```

Reply:
Awesome catch! Done in 59b8865d6e5c768e8513bd8af213e77b9a977ae2